lib/events/playback.go

```diff
@@ -107,6 +102,25 @@ func Export(ctx context.Context, rs io.ReadSeeker, w io.Writer, exportFormat str
 				if err != nil {
 					return trace.ConvertSystemError(err)
 				}
+			case teleport.YAML:
+				_, err := fmt.Println("---")
```

Comment:
This should use Fprintln and write to w. 
```suggestion
				_, err := fmt.Fprintln(w, "---")
```